packages/mui-base/src/PreviewCard/Popup/PreviewCardPopup.types.ts

```diff
@@ -0,0 +1,13 @@
+import type { Side } from '@floating-ui/react';
+import type { BaseUIComponentProps } from '../../utils/types';
+
+export type PreviewCardPopupOwnerState = {
+  open: boolean;
+  side: Side;
+  alignment: 'start' | 'center' | 'end';
```

Comment:
Let's create our own types for both Side and Alignment so we don't expose Floating UI type.